examples/single/bundle/src/App.js

```diff
@@ -38,7 +38,7 @@ class App extends Component {
     );
     this.state = {
       status: "loading",
-      loginIssuer: "https://broker.demo-ess.inrupt.com",
+      loginIssuer: "https://nseydoux.inrupt.net",
```

Comment:
Should this be the demo?

Reply:
No right, it should either be `inrupt.net` or `https://broker.demo-ess.inrupt.com`

Reply:
Fixed in e65ae11bb5e81297c1fbc3e19e0219f4d3b5282d